pkgs/data/fonts/droid/default.nix

```diff
@@ -0,0 +1,59 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "font-droid-${version}";
+  version = "2015-12-09";
+
+  srcs = [
+    (fetchurl {
+      url = "https://github.com/google/fonts/raw/master/apache/droidsans/DroidSans.ttf";
```

Comment:
Actually you could use `fetchFromGitHub` and then copy the versions you want during the install phase